website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -0,0 +1,242 @@
+cd memcached-operator
+operator-sdk init --plugins=ansible --domain=example.com
+```
+
+Now that we have our new project initialized, we need to recreate each of our APIs. 
+Using our API example from earlier (`cache.example.com`), we'll use `cache` for the
+`--group` flag.
+
+For `--version` and `--kind`, we use `spec.versions[0].name` and `spec.names.kind`, respectively.
+
+For each API in the existing project, run:
+
+```sh
+operator-sdk create api \
+    --group=cache \
+    --version=v1 \
+    --kind=Memcached 
+```
+
+Running the above command creates an empty `roles/<kind>`. We can copy over the content of our old `roles/<kind>` to the new one.   
+ 
+### Migrating your Custom Resource samples
+
+Update the CR manifests in `config/samples` with the values of the CRs in your existing project which are in `deploy/crds/<group>_<version>_<kind>_cr.yaml` In our example 
+the `config/samples/cache_v1alpha1_memcached.yaml` will look like:
+
+```yaml
+apiVersion: cache.example.com/v1alpha1
+kind: Memcached
+metadata:
+  name: memcached-sample
+spec:
+  # Add fields here
+  size: 3
+```
+
+### Migrating `watches.yaml
+
+Update the `watches.yaml` file with your `roles/playbooks` and check if you have custom options in the `watches.yaml` file of your existing project. If so, update the new `watches.yaml file to match. 
+
+In our example, we will replace `# FIXME: Specify the role or playbook for this resource.` with our previous role and it will look like:
+
+```yaml
+---
+# Use the 'create api' subcommand to add watches to this file.
+- version: v1alpha1
+  group: cache.example.com
+  kind: Memcached
+  role: memcached 
+# +kubebuilder:scaffold:watch
+```
+
+**NOTE**: Do not remove the `+kubebuilder:scaffold:watch` [marker][marker]. It allows the tool to update the watches file when new APIs are created. 
+
+### Migrating your Molecule tests
+
+If you are using [Molecule][molecule] in your project will be required to port the tests for the new layout.  
+
+See that default structure changed from:
+
+```sh
+├── cluster
+│   ├── converge.yml
+│   ├── create.yml
+│   ├── destroy.yml
+│   ├── molecule.yml
+│   ├── prepare.yml
+│   └── verify.yml
+├── default
+│   ├── converge.yml
+│   ├── molecule.yml
+│   ├── prepare.yml
+│   └── verify.yml
+├── templates
+│   └── operator.yaml.j2
+└── test-local
+    ├── converge.yml
+    ├── molecule.yml
+    ├── prepare.yml
+    └── verify.yml
+
+```
+
+To: 
+
+```
+├── default
+│   ├── converge.yml
+│   ├── create.yml
+│   ├── destroy.yml
+│   ├── kustomize.yml
+│   ├── molecule.yml
+│   ├── prepare.yml
+│   ├── tasks
+│   │   └── foo_test.yml
+│   └── verify.yml
+└── kind
+    ├── converge.yml
+    ├── create.yml
+    ├── destroy.yml
+    └── molecule.yml
+```
+
+And now:
+
+- Use the var `${MOLECULE_PROJECT_DIRECTORY}` instead of `${MOLECULE_EPHEMERAL_DIRECTORY}`
+- Use the var `${KUBECONFIG:-"~/.kube/config"}` instead of `${MOLECULE_EPHEMERAL_DIRECTORY}/kubeconfig` to get the kubeconfig
```

Comment:
This is only true for the `default` scenario (which runs against an existing cluster), the `kind` scenario still uses the ephemeral kubeconfig

Reply:
I will remove this since we have a following up for the molecule. 
We need to check it. 
PS.:  when we scaffold the values now this are the values which appear now. So, would possible to have the old one as well?